libraries/liblmdb/mdb.c

```diff
@@ -3989,7 +3989,7 @@ mdb_txn_commit(MDB_txn *txn)
  * @return 0 on success, non-zero on failure.
  */
 static int ESECT
-mdb_env_read_header(MDB_env *env, MDB_meta *meta)
+mdb_env_read_header(MDB_env *env, MDB_meta *meta, int prev)
```

Comment:
It's not documented anywhere, but our standard practice is to group params with input params first, and output params last. So this should be (MDB_env *env, int prev, MDB_meta *meta).